library/Icinga/Web/RememberMe.php

```diff
@@ -214,7 +208,7 @@ public function setExpiresAt($expiresAt)
      */
     public function authenticate()
     {
-        $authenticated = false;
+        $password = $this->aesCrypt->decryptFromBase64($this->encryptedPassword);
```

Comment:
Please move this directly above line no. 220. You don't need it earlier and only there.

Reply:
I think if the password decryption fails, we don't need the rest.